SETUP.md

```diff
@@ -29,8 +29,6 @@ You can do Code.org development using macOS, Ubuntu, or Windows (running Ubuntu
 
 1. `git lfs pull`
 
-1. `gem install bundler -v 2.3.22 && rbenv rehash`
-
```

Comment:
Arguably this change belongs in the other PR, but this keeps the other PR as small as possible so whatevs.

Reply:
Yeah, this seems reasonable to include in whichever PR; certainly I'm not worried about this breaking anything :smile: